example/ndpiReader.c

```diff
@@ -2680,8 +2680,16 @@ static void setupDetection(u_int16_t thread_id, pcap_t * pcap_handle) {
       exit(-1);
   }
 
-  if(_categoriesDirPath)
-    ndpi_load_categories_dir(ndpi_thread_info[thread_id].workflow->ndpi_struct, _categoriesDirPath);
+  /* Disabled while fuzzing to speed-up init. */
+  if(_categoriesDirPath) {
+#ifndef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
```

Comment:
I understand the reason, but I am not happy about it: we might end up having unit tests with different results, depending on fuzzing being enable or not...
Not sure about other alternatives, though... :(

At very least, I would add something like that (completly untested!):
```
--- a/tests/ossfuzz.sh
+++ b/tests/ossfuzz.sh
@@ -64,4 +64,5 @@ cp fuzz/bd_param.txt $OUT/
 cp fuzz/splt_param.txt $OUT/
 cp fuzz/random_list.list $OUT/
 mkdir -p $OUT/lists
-cp lists/*.list $OUT/lists
+#Ignore a huge list to speed up init time
+find lists/*.list ! -name 100_malware.list -exec cp -t $OUT/lists/ {} +
```
 to avoid slowing down oss-fuzz (we call directly `ndpi_load_categories_dir` from some fuzzers)